apps/custom_auth/forms.py

```diff
@@ -0,0 +1,132 @@
+        del self.fields['username'].widget.attrs['placeholder']
+        del self.fields['email'].widget.attrs['placeholder']
+        del self.fields['password1'].widget.attrs['placeholder']
+        del self.fields['password2'].widget.attrs['placeholder']
+
+    def save(self, request):
+        user = super(DefaultSignupForm, self).save(request)
+        user.get_newsletters = self.cleaned_data['get_newsletters']
+        user.language = get_language()
+        user.save()
+        return user
+
+
+class IgbceSignupForm(DefaultSignupForm):
+    member_number = forms.IntegerField(
+        label=_('IG-BCE member number'),
+        help_text=_('Some helptext.'),
+        max_value=99999999999999999999,
+        min_value=0
+    )
+    birth_date = forms.DateField(
+        label=_('Birth date'),
+        help_text=_('Some helptext.')
+    )
+
+    def validateMemberNumberAndDate(self, member_number, birth_date):
+
+        if (not hasattr(settings, 'IGBCE_NAV_URL') or not hasattr(
+                settings, 'IGBCE_NAV_SECURITYID')):
+            raise forms.ValidationError(
+                "Something is wrong with the setup - please try again later"
+            )
+
+        try:
+            Member.objects.get(member_number=member_number)
+            raise forms.ValidationError(
+                "A member with that number already exists."
+            )
+        except ObjectDoesNotExist:
+            pass
+
+        client = Client('{}'.format(settings.IGBCE_NAV_URL))
+
+        parameters = "{};{}".format(member_number,
+                                    birth_date.strftime("%d.%m.%Y"))
+        connection_parameters = \
+            "ObjectID:0;SecurityID:{};SetSize:0;UnitopProxyVersion:2.0" \
+            .format(settings.IGBCE_NAV_SECURITYID)
+
+        response = client.service.SendRequest(
+            functionName="CALCONNECT_MemberNoBirthDate",
+            functionParameters=parameters,
+            filters="",
+            connectionParameters=connection_parameters
+        )
+
+        result_str = (xmltodict.parse(response)['Response']['ResponseData']
+                      ['Object']['CalConnectorResult'])
+        result = False
+        if result_str == 'true':
+            result = True
+
+        if not result:
+            raise forms.ValidationError(
+                "Your credentials couldn't be connected to an IGBCE member."
+                "Please check your member number and birthdate again."
+            )
+
+    def clean(self):
+        cleaned_data = super().clean()
+        member_number = cleaned_data.get("member_number")
+        birth_date = cleaned_data.get("birth_date")
+
+        self.validateMemberNumberAndDate(member_number, birth_date)
+
+    def save(self, request):
+        user = super(IgbceSignupForm, self).save(request)
+        organisation = Organisation.objects.get(
+            id=settings.SITE_ORGANISATION_ID
+        )
+        additional_info = {
```

Comment:
do I need to tell it, that it has to be json or is putting a dict into a json-field enough?